src/garage/sampler/local_sampler.py

```diff
@@ -0,0 +1,104 @@
+"""Sampler that runs workers in the main process."""
+
+from garage import TrajectoryBatch
+from garage.sampler.sampler import Sampler
+
+
+class LocalSampler(Sampler):
+    """Sampler that runs workers in the main process.
```

Comment:
Is the purpose of saying Local sampler to distinguish from a sampler that uses some parallelization strategy? If so, maybe we can include a little blurb about that?